ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -234,12 +234,13 @@ def ocs_install_verification(
 
     # Verify OSD's are distributed
     if not skip_osd_distribution_check:
-        log.info("Verifying OSD's are distributed evenly across worker nodes")
+        log.info("Verifying OSDs are distributed evenly across worker nodes")
         ocp_pod_obj = OCP(kind=constants.POD, namespace=namespace)
-        osds = ocp_pod_obj.get(selector=constants.OSD_APP_LABEL)['items']
+        osds = ocp_pod_obj.get(selector=constants.OSD_APP_LABEL)
```

Comment:
Not    osds = ocp_pod_obj.get(selector=constants.OSD_APP_LABEL)['items'] ?

Reply:
correct!